.changeset/twenty-eagles-doubt.md

```diff
@@ -0,0 +1,5 @@
+---
+"@mds-core/mds-compliance-service": patch
+---
+
+Refactor ComplianceSnapshotEntityModel and ComplianceViolationEntityModel to directly define types instead of referencing their respective Domain Models to avoid some potentail undesirable TypeORM behavior
```

Comment:
```suggestion
Refactor ComplianceSnapshotEntityModel and ComplianceViolationEntityModel to directly define types instead of referencing their respective Domain Models to avoid some potentially undesirable TypeORM behavior
```

Reply:
Spelling nit